doc/code_style/mom.cpp

```diff
@@ -109,10 +107,8 @@ class HalfLife3 : public ValveGame {
 
 			// You may also start one line later
 			// and then indent with a tab:
-			team.relocate(
-				game_engine->get_team()->get_current_hq_location(),
-				gaben.get_table_location()
-			);
+			team.relocate(game_engine->get_team()->get_current_hq_location(),
+			              gaben.get_table_location());
```

Comment:
that's also possible, but this should remain as-in since it demonstrates the 2 argument indentation methods that are allowed.